go/control/api/grpc_debug.go

```diff
@@ -99,6 +99,10 @@ func (c *debugControllerClient) WaitNodesRegistered(ctx context.Context, count i
 	return c.conn.Invoke(ctx, methodWaitNodesRegistered.FullName(), count, nil)
 }
 
+func (c *debugControllerClient) WaitReady(ctx context.Context, count int) error {
```

Comment:
Why add this here, this seems wrong, the method is for the non-debug part?

Reply:
Oops, forgot to remove this.